malasakit-django/pcari/templates/base_with_links.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
```

Comment:
Why do you have this separate from `base.html`?

Reply:
`base.html` has been changed to not have links in the footnote, to avoid distracting participants. `base_with_links.html` uses template inheritance to override the `footnote` block and re-add the links to the CITRIS site and "About" page.